source-s3/main.go

```diff
@@ -209,8 +209,8 @@ func main() {
 		"title":   "S3 Source Spec",
 		"type":    "object",
 		"required": [
-			"awsAccessKeyId",
-			"awsSecretAccessKey"
```

Comment:
Should we add anything else to the `description` fields for these? Omitting these fields is only really viable if it's completely public, right?

Reply:
Yeah good idea. I added a bit of clarification.